src/cascade/model/object_wrapper.py

```diff
@@ -47,7 +47,7 @@ class ObjectWrapper:
     within those tables into higher-level objects that are then
     easier to reason about.
     """
-    def __init__(self, filename):
+    def __init__(self, filename, unlink=False):
```

Comment:
@gma-uw Can you add some info in the docstring about what `unlink` argument will do?